src/server.h

```diff
@@ -1371,9 +1371,9 @@ typedef struct redisOp {
 /* Defines an array of Redis operations. There is an API to add to this
  * structure in an easy way.
  *
- * redisOpArrayInit();
- * redisOpArrayAppend();
- * redisOpArrayFree();
+ * redisOpArrayInit(void);
+ * redisOpArrayAppend(void);
+ * redisOpArrayFree(void);
```

Comment:
These are not really prototypes. They don't have a return type spelled out either.

If we want prototypes, they're not all `(void)`.

```suggestion
 * int redisOpArrayAppend(redisOpArray *oa, int dbid, robj **argv, int argc, int target);
 * void redisOpArrayFree(redisOpArray *oa);
```

redisOpArrayInit doesn't exist. The only op array we seem to have is `server.also_propagate` which is statically allocated. redisOpArrayFree has a prototype in `server.h` while redisOpArrayAppend is only available within `server.c`.

Reply:
Nice catch, I noticed these were in comments and fixed them, but didn't think through the fact they weren't actually correct :)